.github/workflows/link-rot-checker.yml

```diff
@@ -0,0 +1,25 @@
+name: Link-rot-Checker
```

Comment:
Can we rename to avoid `-`?

Something like:

```yaml
name: Link Checker

name: Broken Link Reporter

name: Automated Link Health Check

name: Scheduled Link Health Check
```